compiler/src/typed/oprint.re

```diff
@@ -767,7 +767,7 @@ and print_out_type_decl = (kwd, ppf, td) => {
         print_list(print_out_constr, ppf => fprintf(ppf, ",@;<0 2>")),
         constrs,
       )
-    | Otyp_open => fprintf(ppf, " = ..")
+    | Otyp_open => fprintf(ppf, " = ...")
```

Comment:
Why did you change this to `...` instead of `..`?

Reply:
I think that was a mistake. Its been corrected